tests/test_wsgi.py

```diff
@@ -75,10 +74,9 @@ def test_environ_headers(self):
         self.assertEqual(environ['HTTP_X_TEST'], '123,456')
         self.assertEqual(environ['SCRIPT_NAME'], 'script')
         self.assertEqual(environ['SERVER_NAME'], 'python.org')
-        self.assertEqual(environ['SERVER_PORT'], '443')
+        self.assertEqual(environ['SERVER_PORT'], '80')
```

Comment:
Oops. I just realized that client may connect to port 123 but say
```GET / HTTP/1.1
Host: example.com:456
```
According to CGI 1.1 in this case SERVER_PORT should be 123.

Ill fix it right now.